src/main/java/com/iexec/sms/tee/session/palaemon/PalaemonSessionService.java

```diff
@@ -74,7 +74,7 @@
     // Compute
     static final String APP_MRENCLAVE = "APP_MRENCLAVE";
     static final String APP_ARGS = "APP_ARGS";
-    static final String APP_PROVIDER_SECRET_PREFIX = "APP_PROVIDER_SECRET_";
+    static final String IEXEC_APP_PROVIDER_SECRET_0 = "IEXEC_APP_PROVIDER_SECRET_";
```

Comment:
The "0" at the end of the variable name is confusing.

Reply:
Thanks, that's a bad copy/paste.